src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/runtime.FreeBSD.Microsoft.NETCore.Runtime.CoreCLR.props

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
-    <NativeBinary Include="$(BinDir)libcoreclr.so" />
+    <NativeBinary Include="$(BinDir)libcoreclr.so" LinkerUsage="runtime" />
     <NativeBinary Condition="'$(_PlatformDoesNotSupportEventTrace)' != 'true'" Include="$(BinDir)libcoreclrtraceptprovider.so" />
     <NativeBinary Include="$(BinDir)libdbgshim.so" />
-    <NativeBinary Include="$(BinDir)libmscordaccore.so" />
-    <NativeBinary Include="$(BinDir)libmscordbi.so" />
+    <NativeBinary Include="$(BinDir)libmscordaccore.so" LinkerUsage="diagnostic" />
+    <NativeBinary Include="$(BinDir)libmscordbi.so" LinkerUsage="diagnostic" />
     <NativeBinary Include="$(BinDir)libsos.so" />
     <NativeBinary Condition="'$(_PlatformDoesNotSupportSosPlugin)' != 'true'" Include="$(BinDir)libsosplugin.so" />
     <NativeBinary Include="$(BinDir)System.Globalization.Native.so" />
```

Comment:
This isn't my area of expertise, but don't you need System.Globalization.Native also?